query/src/test/java/org/infinispan/query/dsl/embedded/impl/LuceneTransformationTest.java

```diff
@@ -539,21 +539,21 @@ public void testNotLikeQuery() {
    public void testIsNullQuery() {
       assertGeneratedLuceneQuery(
             "select e from org.infinispan.query.dsl.embedded.impl.model.Employee e where e.name IS null",
-            "+(-ConstantScore(__HSEARCH_field_names:name) #*:*)");
+            "+(-ConstantScore(NormsFieldExistsQuery [field=name]) #*:*)");
```

Comment:
This change is only necessary because the previous parsing of `@Field` was buggy and ignored the analyzer.

Now that we correctly detect the analyzer, we create a full-text-field as we should, thus Hibernate Search knows any field with a value has norms, and it can use this more efficient way of detecting the presence of a field.